org_status/org_hosts/__init__.py

```diff
@@ -22,6 +22,10 @@ def print_status(self, repo_url):
     def process_repository(self, repo):
         raise NotImplementedError()
 
+    @staticmethod
```

Comment:
Please make this a classmethod.

Reply:
Since the current terminology used also calls repo build statuses as status, please change this to `get_host_status()`.